cockroachcloud/managing-cmek.md

```diff
@@ -9,7 +9,10 @@ docs_area: manage.security
 
 This page shows how to enable [Customer-Managed Encryption Keys (CMEK)](cmek.html) for a {{ site.data.products.dedicated }} cluster.
 
-To follow this procedure requires admin access to your {{ site.data.products.dedicated }} organization, and the ability to create and manage identity and access management (IAM) and key management (KMS) services in your organization's cloud, i.e., your Google Cloud Platform (GCP) project or Amazon Web Services (AWS) account.
+**Prerequisites**:
+
+- {% include cockroachcloud/cluster-operator-prereq.md %}
+- Sufficient permissions to create and manage identity and access management (IAM) and key management (KMS) services in your organization's cloud, i.e., your Google Cloud Platform (GCP) project or Amazon Web Services (AWS) account.
```

Comment:
I know you didn't change this but in the "Before you begin" section of this page, the first bullet there is an errant `]` after "private cluster", if you change anything else in this PR, perhaps you could remove?